grunt/config/replace.js

```diff
@@ -62,7 +62,18 @@ module.exports = function (grunt, options) {
           grunt.log.writeln('WARNING: xAPI activityID has not been set');
         }
       }
-      
+
+      // Shim to preserve the 'adapt_scorm' identifier.
+      if (configJson.hasOwnProperty('_spoor')) {
+        if (comfigJson._spoor._advancedSettings) {
```

Comment:
typo 'comfig' here and on all the following lines

Reply:
How embarrassing.